src/lib/y2storage/filesystems/blk_filesystem.rb

```diff
@@ -114,6 +114,21 @@ def plain_blk_devices
         blk_devices.map(&:plain_device)
       end
 
+      # Block device base name
+      #
+      # When the filesystem is a non-multidevice, this method simply returns the base
+      # name of the block device (e.g., "sda1"). And for multidevice ones, it only returns
+      # the base name of the first block device plus a "+" symbol to indicate that the
+      # filesystem is multidevice (e.g., "sda1+").
+      #
+      # @return [String]
+      def blk_device_basename
+        info = blk_devices.first.basename
+        info << "+" if multidevice?
+
```

Comment:
Can you check the value of blk_devices.first.basename right after this `<<`?

Reply:
Are you worried by the reference? Take into account that `BlkDevice#basename` always returns a new String object, so no problem.